nexus_msgs/nexus_transporter_msgs/msg/Destination.msg

```diff
@@ -0,0 +1,14 @@
+# Name of the destination
+string name
+
+# Action constants for the action field.
+int32 ACTION_UNDEFINED=0
+int32 ACTION_TRANSIT=1
+int32 ACTION_PICKUP=2
+int32 ACTION_DROPOFF=3
+
+# The type of action to perform at the destination.
+int32 action
```

Comment:
we probably won't need so many action types in the future, we can use an uint8 for this field